experiments/is_my_project_on_track.md

```diff
@@ -0,0 +1,15 @@
+- are you hacking?
```

Comment:
The idea behind this document is valuable: Shield Studies can take on the order of months from start to finish. Here is a timeline breakdown and dependency graph of things you should be thinking about and when throughout the course of developing a Shield study.